### ISSUE TITLE
[BUG] The last version of PIXI will break the mouse & touch event.

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#3662]
My event listener like this : 
```
window.addEventListener('mousedown', function(){
 alert("mousedown');
},false);
```
When I import pixi.js , the listener never be fired. 
If remove pixi.js or use old version , everything will be OK.

I think the PR https://github.com/pixijs/pixi.js/pull/3546 makes this bug.

**Comment by finscn** [comment]
I found the key : 
```
    InteractionManager.prototype.onPointerDown = function onPointerDown(originalEvent) {
        var events = this.normalizeToPointerData(originalEvent);

        /**
         * No need to prevent default on natural pointer events, as there are no side effects
         * Normalized events, however, may have the double mousedown/touchstart issue on the native android browser,
         * so still need to be prevented.
         */

        // Guaranteed that there will be at least one event in events, and all events must have the same pointer type

        if (this.autoPreventDefault && (events[0].pointerType === 'touch' || events[0].pointerType === 'mouse')) {
            originalEvent.preventDefault();
        }

```

the default value of autoPreventDefault  is true ,
So the onPointerDown will break other event.

**Comment by finscn** [comment]
@starakaj  , in your PR , mouse event  & touch event  both use pointer event.
And the autoPreventDefault of pointer event is true.

**Comment by finscn** [comment]
InteractionManager has a parameter `autoPreventDefault` , 
But there is a code will be called when load PIXI
```
core.WebGLRenderer.registerPlugin('interaction', InteractionManager);
```

User can't set the `autoPreventDefault`

**Comment by staff0rd** [comment]
@finscn does `renderer.plugins.interaction.autoPreventDefault = false` solve your problem?

**Comment by finscn** [comment]
@staff0rd  , yes , it could. But , you see this : 
```
        /**
         * No need to prevent default on natural pointer events, as there are no side effects
         * Normalized events, however, may have the double mousedown/touchstart issue on the native android browser,
         * so still need to be prevented.
         */
```

I think if I set `autoPreventDefault = false` , there may be a new problem.

**Comment by andrewstart** [comment]
If you are listening for mouse events, and are on a touch device, then you need to set `autoPreventDefault` to false. The browser will fire the touchevent, followed by the mouse event on a short delay, as it always has. That behavior should not have changed in any recent PRs.

If you aren't getting the mousedown events in a desktop environment, then it sounds like it should be a different cause.

**Comment by finscn** [comment]
@andrewstart  ， my device is mac book pro . no touch event.
 I test touch event on android.

**Comment by staff0rd** [comment]
@finscn can you get a [pen](https://codepen.io) up against [dev](//d157l7jdn8e5sf.cloudfront.net/dev/pixi.js) so we can test fixes?

> Maybe we should make a simple way to do that.

That's the way to set values against plugins atm.  Open another issue if you've got a better idea ;)

**Comment by finscn** [comment]
@staff0rd  ，sorry ， I don't  know how to use codepen .
And in our country ( China) ,  the codepen is hard to be visited , so even I can't learn how to use it.

**Comment by staff0rd** [comment]
@finscn ah ok.  Are there any other javascript sandboxes that are accessible through the GFW?  Like jsfiddle or jsbin for example?

**Comment by finscn** [comment]
```
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width">
  <title>JS Bin</title>
  <script src="https://d157l7jdn8e5sf.cloudfront.net/dev/pixi.js"></script>

</head>
<body>
  <canvas id='canvas' style="background:#fff0e0;"></canvas>
<script>
  document.addEventListener('mousedown', function(){
       alert('mousedown')
  },false) ;
  
  var canvas;
  var renderer;
  window.onload = function(){
     canvas = document.getElementById('canvas');
     renderer = PIXI.autoDetectRenderer(512, 256, { view: canvas, ratio: 1 });
  };
</script>
</body>
</html>
```

@staff0rd  , you could paste the code into  http://jsbin.com/ 

click the canvas, there will no effect.

**Comment by andrewstart** [comment]
Ah ha, I've figured it out, and added a fix to my PR: #3331. The browser was sending a `PointerEvent` for the mouse down event, and we were incorrectly calling `preventDefault()` on it, which prevented a regular mouse event from firing.
